server/index.js

```diff
@@ -94,6 +95,26 @@ app.init = function({
         filter: contentType => !contentType.startsWith('text/event-stream'),
       })
     )
+    .use(async function(ctx, next) {
+      if (
+        process.env.ENABLE_AUTH &&
+        process.env.AUTH_TYPE === 'ADMIN_JWT' &&
+        process.env.AUTH_ADMIN_JWT_PRIVATE_KEY
```

Comment:
if you can move these variables to the init args, it will allow users to either use the env variables OR params to init function.